salt/modules/highstate_doc.py

```diff
@@ -655,22 +655,30 @@ def proccesser_markdown(lowstate_item, config, **kwargs):
     if s.get("watch"):
         requisites += "run or update after changes in:\n"
         for w in s.get("watch", []):
-            requisites += _format_markdown_requisite(w.items()[0][0], w.items()[0][1])
+            requisites += _format_markdown_requisite(
+                next(iter(w.items()))[0], next(iter(w.items()))[1],
```

Comment:
`*next(iter(w.values()))`
the same is below here

Reply:
done